adam-python/bdgenomics/adam/rdd.py

```diff
@@ -43,6 +43,32 @@ def __init__(self, jvmRdd, sc):
         self.sc = sc
 
 
+    def cache(self):
+        """
+        Caches underlying RDD in memory.
+        """
+
+        return self._replaceRdd(self._jvmRdd.cache())
+
+
+    def persist(self, sl):
+        """
+        Persists underlying RDD in memory.
+
+        :param sl new StorageLevel
+        """
+
+        return self._replaceRdd(self._jvmRdd.persist(self.sc._jvm.org.apache.spark.api.java.StorageLevels.create(sl.useDisk, \
```

Comment:
This line is really long. Can you split it out as:

```
jsl = self.sc._jvm.org.apache.spark.api.java.StorageLevels.create(sl.useDisk,
  sl.useMemory,
  sl.useOffHeap,
  sl.deserialized,
  sl.replication)

return self._replaceRdd(self._jvmRdd.persist(jsl))
```

Indented parameters should align with the [opening delimiter](https://www.python.org/dev/peps/pep-0008/#indentation).